render/http.go

```diff
@@ -258,7 +258,7 @@ func prettyPrintHTTPTest(test api.HTTPTest, variables map[string]string) string
 	}
 	if test.BodyContains != nil {
 		interpolated := checks.InterpolateVariables(*test.BodyContains, variables)
-		return fmt.Sprintf("Expecting JSON body to contain: %s", interpolated)
+		return fmt.Sprintf("Expecting body to contain: %s", interpolated)
```

Comment:
Also remove "JSON" from BodyContainsNone string on line 265? I can add this to my changes for new combined lesson type.

Reply:
Yeah if you can just throw that in that would be great, didn't see that one